.github/NEW_COMPONENT_PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,79 @@
+<!--
+Thank you for contributing! Please follow the steps below to help us process your PR quickly.
+
+- 📝 Use a meaningful title for the pull request and include the name of the package modified.
+- 📓 Ensure each of your commit messages adhere to the conventional commit specification.
+- ✅ Add or edit tests to reflect the change (run `yarn test`).
+- 🙏 Please review your own PR to check for anything you may have missed.
+-->
+
+[REPLACE_WITH_JIRA_TICKET_NUMBER](https://oktainc.atlassian.net/browse/REPLACE_WITH_JIRA_TICKET_NUMBER)
+
+## Summary
+
+<!--
+  Add a description with these talking points:
+  1. Figma link if applicable.
+  2. A brief description of the work and why it was done in this particular way.
+-->
+
+## Component checklist
+
+<!-- Pease track and mark relevant items as part of the component development process -->
+
+### Code structure
+
+- [ ] Organize code using a modular component architecture.
+- [ ] Use semantic HTML for clear and meaningful structure.
+- [ ] Break down complex components into smaller, reusable parts.
+
+### Accessibility implementation
+
+- [ ] Apply ARIA attributes to indicate roles and relationships.
+- [ ] Test focus management and ensure interactive elements are keyboard accessible.
+- [ ] Include skip-to-content links for screen reader users.
+- [ ] Provide visual alternatives for auditory elements.
+
+### Internationalization (i18n) implementation
+
+- [ ] Separate text content from the component's code for translation.
+- [ ] Ensure the component renders correctly when language changes.
+
+### Responsive implementation
+
+- [ ] Use CSS media queries to define breakpoints and apply styles.
+- [ ] Test fluid layouts and resizing behavior at each breakpoint.
+- [ ] Optimize images for different screen resolutions.
```

Comment:
I know this is part of the component workflow checklist but, now that I think about it, we probably don't need this. Odyssey shouldn't be serving any images, I wouldn't think

Reply:
@bryancunningham-okta makes sense, I updated the text to say "as needed". The comment in the template also mentions to use of relevant parts. Do you think that would help avoid confusion?